src/servicebus/Elsa.AzureServiceBus/Activities/ServiceBusSignalReceived.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Elsa;
+using Elsa.Attributes;
+using Elsa.Extensions;
+using Elsa.Results;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elsa.AzureServiceBus.Activities
+{
+    [ActivityDefinition(
+        Category = "Azure",
+        Description = "Wait for an azure message to be receid to start the activity",
+        //Icon = "fas fa-traffic-light", 
+        Outcomes = new[] { OutcomeNames.Done }
+     )]
+    public class ServiceBusSignalReceived : Activity
+    {
+        public const string INPUT_VARIABLE_NAME = "AzureServiceBusMessage";
```

Comment:
Please use the established naming conventions. In this case, public constants are named like regular public properties, so: `InputVariableName`. I've seen this in the other activity class as well.